base-theme/layouts/partials/footer.html

```diff
@@ -1,112 +1,114 @@
 <footer id="footer-container">
-  <div class="row pb-4 mx-0 justify-content-between">
-    <div>
-      <a
-        id="open-learning-logo"
-        href="https://openlearning.mit.edu/"
-        target="_blank"
+  <div id="home-footer">
```

Comment:
There was a lot of confusing inconsistency in formatting here so I ran `prettier`